### DISCUSSION TITLE
Token login for lovelace dashboard (kiosk mode)

### DISCUSSION THREAD

**Discussion by andyb2000** [home-assistant/frontend#19948] [Dashboard]
I'd like to be able to display certain dashboards on screens (Tv's with smart web browsers) around various centres where HA is providing integration with heating controllers.

These can't use IP authentication as they all go through a cloudflare ssl proxy (so appear from the same IP) and also cannot login using a username+password as there is no facility to enter details.

Therefore, what I'm looking for is a URL that can be passed a token (probably Long-Lived is the right way) and dashboard to display. The authentication is the bit that I'm requesting, a parameter (GET) to allow the browser to login and display the dashboard.

**Comment by jonnywright** [comment]
I had a very similar thought yesterday. With many modern cars now incorporating web-browsers (mine included), I've built an in-vehicle specific dashboard. It was logged out yesterday and had to re-enter credentials, which wasn't pleasant on that type of screen. I thought to myself "It'd be great if I could login via. QR code, or authenticate with a OTP on a more user friendly device".
